distrobuilder/main.go

```diff
@@ -125,7 +125,7 @@ func main() {
 
 	app := &cobra.Command{
 		Use:   "distrobuilder",
-		Short: "System container and VM image builder for LXC and LXD",
+		Short: "System container and VM image builder for LXC and Incus",
```

Comment:
Any objection to have the test say "LXD/Incus" or "Incus/LXD"

Reply:
Not at all. I'll change it.